test/integration/targets/aws_lambda/tasks/main.yml

```diff
@@ -139,8 +139,7 @@
       lambda:
         name: "{{lambda_function_name}}"
         runtime: "nodejs10.x"
-# Can only be uncommented once the ansible_lambda_role has permission to call PutTraceSegments on XRAY
-#        tracing_mode: 'Active'
+        tracing_mode: 'Active'
```

Comment:
Please add the assertion I had above that PassThrough is set as the default.

Reply:
Done